src/GraphQL/Types/Composite/AutoRegisteringOutputHelper.cs

```diff
@@ -99,12 +99,6 @@ public static void BuildFieldType(
         {
             throw new ArgumentOutOfRangeException(nameof(memberInfo), "Member must be a field, property or method.");
         }
-        if (buildMemberInstanceExpressionFunc == null)
-        {
-            // interface types do not set resolvers
-            fieldType.Resolver = null;
-            fieldType.StreamResolver = null;
-        }
```

Comment:
Moved to AutoRegisteringInterfaceGraphType.CreateField so that the attributes (specifically IsPrivate) gets applied first